pyi.py

```diff
@@ -308,6 +308,9 @@ def _is_object(node: ast.AST | None, name: str, *, from_: Container[str]) -> boo
 _is_BaseException = partial(_is_object, name="BaseException", from_={"builtins"})
 _is_TypeAlias = partial(_is_object, name="TypeAlias", from_=_TYPING_MODULES)
 _is_NamedTuple = partial(_is_object, name="NamedTuple", from_=_TYPING_MODULES)
+_is_deprecated = partial(
```

Comment:
This name is a little confusing. I considered `_is_at_deprecated` or `_is_deprecated_decorator`, but not sure that's much better. Note that we already have `_is_final` and `_is_unused` which have similar issues.